include/deal.II/distributed/tria.h

```diff
@@ -373,6 +373,10 @@ namespace parallel
 
       /**
        * Implementation of the same function as in the base class.
+       *
+       * @note This function can be use to copy a serial Triangulation to a
+       * parallel::distributed::Triangulation but only if the serial
+       * Triangulation has never been refined.
```

Comment:
Interesting. I hadn't realized. Do we have an assertion that triggers when one tries?